app/src/org/commcare/android/view/TabbedDetailView.java

```diff
@@ -183,5 +196,30 @@ public int getCurrentTab() {
     public int getTabCount() {
         return mViewPager.getAdapter().getCount();
     }
-    
+
+    //region Private classes
+
+    private class DefaultEDVModifier implements EntityDetailAdapter.EntityDetailViewModifier, Parcelable {

```

Comment:
I'm only using Parcelable to automatically save the odd/even row state between screen layout changes (like turning the phone to the side)